control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -3249,15 +3249,36 @@ func (r *HostedControlPlaneReconciler) reconcileCoreIgnitionConfig(ctx context.C
 		return nil
 	}
 
-	imageDigestMirrorSet := globalconfig.ImageDigestMirrorSet()
-	if err := globalconfig.ReconcileImageDigestMirrors(imageDigestMirrorSet, hcp); err != nil {
-		return fmt.Errorf("failed to reconcile image content policy: %w", err)
+	version, err := semver.Parse(releaseImageProvider.Version())
+	if err != nil {
+		return fmt.Errorf("failed to determine release image version: %w", err)
 	}
 
-	if _, err := createOrUpdate(ctx, r, imageContentPolicyIgnitionConfig, func() error {
-		return ignition.ReconcileImageSourceMirrorsIgnitionConfig(imageContentPolicyIgnitionConfig, p.OwnerRef, imageDigestMirrorSet)
-	}); err != nil {
-		return fmt.Errorf("failed to reconcile image content source policy ignition config: %w", err)
+	// ImageDigestMirrorSet is only applicable for release image versions greater than or equal to 4.13
+	if version.Minor >= 13 {
```

Comment:
Just a comment for myself... this could be done implicitly by only backporting the change to the 4.13 branch. However, it complicates backporting to 4.12 because it's one more thing we need to be aware of not to backport.